src/STEMMUS_SCOPE.m

```diff
@@ -128,7 +371,7 @@
 %% 4. input data
 % the current stemmus-scope does not support useXLSX=0
 if useXLSX == 0
-    X                           = textread([path_input parameter_file{3}], '%s'); %#ok<DTXTRD>
+    X                           = textread([path_input parameter_file{3}], '%s'); % ok<DTXTRD>
```

Comment:
I think the `#` should stay: it suppresses Matlab's code analyzer. https://nl.mathworks.com/matlabcentral/answers/483637-what-does-the-string-ok-nasgu-mean-in-the-code-editior#answer_394861